backend/remote/backend_test.go

```diff
@@ -126,6 +126,42 @@ func TestRemote_config(t *testing.T) {
 	}
 }
 
+// TestRemote_cached_config is similar to the test above, but we mimic loading a
+// cached backend config by skipping the validate call
+func TestRemote_config_cached(t *testing.T) {
```

Comment:
I think the convention for naming tests is using only 1 underscore in a name. So that would mean this test should be called something like: `TestRemote_cachedConfig`